style/americana.js

```diff
@@ -155,11 +182,15 @@ var map = new maplibregl.Map({
   zoom: 4, // starting zoom
   attributionControl: false,
 });
+
+map.on("styleimagemissing", function (e) {
+  Shield.missingIconLoader(map, e);
```

Comment:
Where is `Shield` defined?

Reply:
Whoops, this was an accidental import from my highway shields branch.